stories/components/person/person.style.js

```diff
@@ -122,7 +123,7 @@ export const customCSSProperties = () => html`
     dndPerson.personPresence = dnd;
   </script>
 
-  <mgt-person class="person" person-query="me" view="fourlines" id="online" show-presence></mgt-person>
+  <mgt-person person-query="me" view="fourlines" id="online" show-presence></mgt-person>
```

Comment:
```suggestion
  <mgt-person class="person" person-query="me" view="fourlines" id="online" show-presence></mgt-person>
```

Required since we're using classes over element tags in CSS